api/app/services/demonstration/data_loader.rb

```diff
@@ -70,6 +77,25 @@ def create_pages
       end
     end
 
+    def ensure_settings
+      settings = Settings.instance
+      ENV_SETTINGS.each do |key, values|
+        values.each do |value|
+          env_setting = "MANIFOLD_SETTING_#{key}_#{value}".upcase
+          settings[key][value] = settings[key][value] ||= ENV[env_setting]
+          @logger.info("Setting updated: #{key}: #{value}".green)
```

Comment:
I took this out, but I wasn't sure if you wanted it in there and give context based feedback.  I can add it back in that manner if you'd rather.